src/Hazelcast.Net.Tests/Resources/Options/HazelcastFailoverOptions.json

```diff
@@ -0,0 +1,299 @@
+{
```

Comment:
Not related to this, but there is an empty file called `src/Hazelcast.Net.Tests/Resources/HazelcastFailoverOptions.txt`